types/request.d.ts

```diff
@@ -30,6 +30,8 @@ export interface FastifyRequest<
   hostname: string;
   url: string;
   method: string;
+  routerUrl: string;
+  routerMethod: string;
```

Comment:
@Eomm You suggested exposing router method; are there any cases when you expect router method to differ from request method?